README.md

```diff
@@ -114,3 +114,14 @@ In pipelines, there're two ways to construct this binding:
         sh 'az login --service-principal -u $CLIENT_ID -p $CLIENT_SECRET -t $TENANT_ID'
     }
     ```
+
+## Reporting bugs and feature requests
+
+We use [Jenkins JIRA](https://issues.jenkins-ci.org/) to record all bugs and feature requests. Please follow beblow steps to create your own issues.
+
+1. Search in Jira to see if the issue was existed already.
+2. Create a new Jenkins account [here](https://accounts.jenkins.io/signup) if you don't have one.
+3. Browse to [Jenkins JIRA](https://issues.jenkins-ci.org/) and login with your account.
+4. Create a new issue with the component `azure-credentials-plugin` .
```

Comment:
I think step 2 and step 3 is not necessary. It's Jira flow, Jira will guide user to go through signup/signin flow. Add it here will miss our focus.  for example, in a normal github readme filling issue description, it will not show step of signin or sign up. what's your thoughts? @gavinfish @yuwzho

Reply:
agree